<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmailListEditorDialog</class>
 <widget class="QDialog" name="EmailListEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Email List Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Email List:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="emaillistWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="isWrapping" stdset="0">
        <bool>false</bool>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addEmailAddressButton">
       <property name="text">
        <string>Add An Email Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tipsLabel">
       <property name="text">
        <string>Tips: You can double-click the email address in the edit list, or click the email to pop up the option menu.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionDelete_Selected_Email_Address">
   <property name="text">
    <string>Delete Selected Email Address</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
